client/src/shared/components/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable multiline-ternary */
+import { useRouter } from 'next/router';
+import React, { Fragment } from 'react';
+
+interface BreadcrumbsProps {
+  excludePath: string[];
+}
```

Comment:
- I think props is not that helpful in actual operation. 
- could try the implementation below instead.
```
paths=[{
	{
		text: 'Profile'
		url: '/profile'
	},
	{
		text: 'User Information'
		url: '/profile/information'
	},
	{
		text: 'Edit Information',
		url: '/profile/information/edit'
	},
}];
currentPath="/"


// so if currentPath="/profile/information". It should displays a breadcrumb as below
Profile > User Information

// if currentPath="profile/information/edit". display
Profile > User Information > Edit Information
```


**Important** for demo. no need to add extra page. Just showing that it can render the right breadcrumb is enough.